sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -59,33 +61,31 @@ def __init__(self, endpoint, credential, **kwargs):
         self._refresh_token = None # type: Optional[str]
         self._expiration_time = 0 # type: float
 
-    def get_acr_access_token(self, challenge, **kwargs):
-        # type: (str, Any) -> Optional[str]
+    def get_acr_access_token(self, challenge: str, **kwargs) -> Optional[str]:
         parsed_challenge = _parse_challenge(challenge)
         refresh_token = self.get_refresh_token(parsed_challenge["service"], **kwargs)
         return self.exchange_refresh_token_for_access_token(
             refresh_token, service=parsed_challenge["service"], scope=parsed_challenge["scope"], **kwargs
         )
 
-    def get_refresh_token(self, service, **kwargs):
-        # type: (str, Any) -> str
+    def get_refresh_token(self, service: str, **kwargs) -> str:
         if not self._refresh_token or self._expiration_time - time.time() > 300:
             self._refresh_token = self.exchange_aad_token_for_refresh_token(service, **kwargs)
             self._expiration_time = _parse_exp_time(self._refresh_token)
         return self._refresh_token
 
-    def exchange_aad_token_for_refresh_token(self, service, **kwargs):
-        # type: (str, Any) -> str
+    def exchange_aad_token_for_refresh_token(self, service: str, **kwargs) -> str:
         refresh_token = self._client.authentication.exchange_aad_access_token_for_acr_refresh_token(
             grant_type=PostContentSchemaGrantType.ACCESS_TOKEN,
             service=service,
-            access_token=self._credential.get_token(*self.credential_scopes).token,
+            access_token=self._credential.get_token(self.credential_scopes).token,
```

Comment:
The `*` should not be removed.
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/credentials.py#L27